lib/compile.coffee

```diff
@@ -199,7 +200,7 @@ compile = (options) ->
         # Strip out any filters (e.g. ng-repeat="foo in bar | limitTo: 10")
         # and split by whitespace and compact the result (remove any empty
         # strings in the list) as well as the 'track by' option in angular
-        repeatExpSplit = _.compact repeatExp
+        repeatExpSplit = repeatExp
```

Comment:
@steve8708 I have not fully tested the compiled JS in our project, so you may want to hold off merging until I do that.